DataFormats/Scouting/interface/Run3ScoutingMuon.h

```diff
@@ -0,0 +1,305 @@
+        nValidStripHits_(nValidStripHits),
+        nPixelLayersWithMeasurement_(nPixelLayersWithMeasurement),
+        nTrackerLayersWithMeasurement_(nTrackerLayersWithMeasurement),
+        trk_chi2_(trk_chi2),
+        trk_ndof_(trk_ndof),
+        trk_dxy_(trk_dxy),
+        trk_dz_(trk_dz),
+        trk_qoverp_(trk_qoverp),
+        trk_lambda_(trk_lambda),
+        trk_pt_(trk_pt),
+        trk_phi_(trk_phi),
+        trk_eta_(trk_eta),
+        trk_dxyError_(trk_dxyError),
+        trk_dzError_(trk_dzError),
+        trk_qoverpError_(trk_qoverpError),
+        trk_lambdaError_(trk_lambdaError),
+        trk_phiError_(trk_phiError),
+        trk_dsz_(trk_dsz),
+        trk_dszError_(trk_dszError),
+        trk_qoverp_lambda_cov_(trk_qoverp_lambda_cov),
+        trk_qoverp_phi_cov_(trk_qoverp_phi_cov),
+        trk_qoverp_dxy_cov_(trk_qoverp_dxy_cov),
+        trk_qoverp_dsz_cov_(trk_qoverp_dsz_cov),
+        trk_lambda_phi_cov_(trk_lambda_phi_cov),
+        trk_lambda_dxy_cov_(trk_lambda_dxy_cov),
+        trk_lambda_dsz_cov_(trk_lambda_dsz_cov),
+        trk_phi_dxy_cov_(trk_phi_dxy_cov),
+        trk_phi_dsz_cov_(trk_phi_dsz_cov),
+        trk_dxy_dsz_cov_(trk_dxy_dsz_cov),
+        trk_vx_(trk_vx),
+        trk_vy_(trk_vy),
+        trk_vz_(trk_vz),
+        trk_hitPattern_(trk_hitPattern),
+        vtxIndx_(std::move(vtxIndx)) {}
+  //default constructor
+  Run3ScoutingMuon()
+      : pt_(0),
+        eta_(0),
+        phi_(0),
+        m_(0),
+        type_(0),
+        charge_(0),
+        normalizedChi2_(0),
+        ecalIso_(0),
+        hcalIso_(0),
+        trackIso_(0),
+        nValidStandAloneMuonHits_(0),
+        nStandAloneMuonMatchedStations_(0),
+        nValidRecoMuonHits_(0),
+        nRecoMuonChambers_(0),
+        nRecoMuonChambersCSCorDT_(0),
+        nRecoMuonMatches_(0),
+        nRecoMuonMatchedStations_(0),
+        nRecoMuonExpectedMatchedStations_(0),
+        recoMuonStationMask_(0),
+        nRecoMuonMatchedRPCLayers_(0),
+        recoMuonRPClayerMask_(0),
+        nValidPixelHits_(0),
+        nValidStripHits_(0),
+        nPixelLayersWithMeasurement_(0),
+        nTrackerLayersWithMeasurement_(0),
+        trk_chi2_(0),
+        trk_ndof_(0),
+        trk_dxy_(0),
+        trk_dz_(0),
+        trk_qoverp_(0),
+        trk_lambda_(0),
+        trk_pt_(0),
+        trk_phi_(0),
+        trk_eta_(0),
+        trk_dxyError_(0),
+        trk_dzError_(0),
+        trk_qoverpError_(0),
+        trk_lambdaError_(0),
+        trk_phiError_(0),
+        trk_dsz_(0),
+        trk_dszError_(0),
+        trk_qoverp_lambda_cov_(0),
+        trk_qoverp_phi_cov_(0),
+        trk_qoverp_dxy_cov_(0),
+        trk_qoverp_dsz_cov_(0),
+        trk_lambda_phi_cov_(0),
+        trk_lambda_dxy_cov_(0),
+        trk_lambda_dsz_cov_(0),
+        trk_phi_dxy_cov_(0),
+        trk_phi_dsz_cov_(0),
+        trk_dxy_dsz_cov_(0),
+        trk_vx_(0),
+        trk_vy_(0),
+        trk_vz_(0),
+        trk_hitPattern_(reco::HitPattern()),
```

Comment:
Copy-constructing from default-constructed temporary is not necessary, implicit call to default constructor is sufficient
```suggestion
```